### PR TITLE
Shoira's file

### PR DESCRIPTION
**Pull request by shoirka**
Welcome class is created. Please review and approve.

### CODE REVIEW

File: src/TeamWork/TeamWork2.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package TeamWork;
+
+public class TeamWork2 {
```
Comment (marat-aksiev) [inline src/TeamWork/TeamWork2.java:3]: good

**Review by marat-aksiev — COMMENTED** [review]
nice

**Comment by shoirka** [comment]
thanks

**Comment by MarieKiri** [comment]
All good! Thank you

**Pull request merged**